enterprise/models.py

```diff
@@ -850,13 +858,31 @@ class EnterpriseCustomerCatalog(TimeStampedModel):
             "included in the catalog."
         )
     )
+    enabled_course_modes = JSONField(
+        default=mode_slugs,
```

Comment:
This should be `default= COURSE_MODE_SORT_ORDER`. Import `COURSE_MODE_SORT_ORDER` from [here](https://github.com/edx/edx-enterprise/blob/master/enterprise/constants.py#L30). Add `'credit'` to `COURSE_MODE_SORT_ORDER`.

Reply:
@douglashall FYI, I moved the `json.dumps()` from JSONField to callback --- `mode_slugs` here because the build was failing on Travis with Python=3.5. I was getting the JSON serializer error, I think due to the JSONField encoder. Kindly take a look, If we are good then we can merge it. 

-- Now build is also passing with `tox` now.